PhoneNumberKit/UI/TextField.swift

```diff
@@ -180,6 +205,52 @@ open class PhoneNumberTextField: UITextField, UITextFieldDelegate {
         super.delegate = self
     }
 
+    open func exampleNumber(for countryCode: String) -> String {
+        let exampleNationalNumber = phoneNumberKit.metadata(for: currentRegion)?.mobile?.exampleNumber ?? "12345678"
+        do {
+            let phoneNumber = try phoneNumberKit.parse(exampleNationalNumber, withRegion: currentRegion, ignoreType: true)
+            return phoneNumberKit.format(phoneNumber, toType: .international, withPrefix: true)
+        } catch {
+            print("Failed to parse example phone number in PhoneNumberKit")
+        }
+        return exampleNationalNumber
+    }
+
+    func internationalPrefix(for countryCode: String) -> String? {
+        guard let countryCode = phoneNumberKit.countryCode(for: currentRegion)?.description else { return nil }
+        return "+" + countryCode
+    }
+
+    open func updateFlag() {
+        guard withFlag else { return }
+        let flagBase = UnicodeScalar("🇦").value - UnicodeScalar("A").value
+
+        let flag = currentRegion
+            .uppercased()
+            .unicodeScalars
+            .compactMap({ UnicodeScalar(flagBase + $0.value)?.description })
+            .joined()
+
+        flagButton.setTitle(flag + " ", for: .normal)
+        let fontSize = (font ?? UIFont.preferredFont(forTextStyle: .body)).pointSize
+        flagButton.titleLabel?.font = UIFont.systemFont(ofSize: fontSize)
+    }
+
+    open func updatePlaceholder() {
+        guard withExamplePlaceholder else { return }
+        example = exampleNumber(for: currentRegion)
```

Comment:
if you move `exampleNumber(for:)` to the PhoneNumberKit class, then this ends up being

`placeholderNumber = phoneNumberKit.exampleNumber(for: currentRegion)`